recipe/meta.yaml

```diff
@@ -6,8 +6,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nmma-{{ version }}.tar.gz
-  sha256: e22e0136ab4179576b0986a8929f12556d7cfc7d6bcdd54a1378bb6e7c1513d9
+  url: https://github.com/nuclear-multimessenger-astronomy/nmma/archive/refs/heads/main.tar.gz
```

Comment:
Please use a tag. This is untraceable as main changes. (If you are debugging before releases, it is fine, but please don't merge it before issuing a tag or publishing a new version on PyPI.)

Reply:
We already published a tag on PyPi a week ago. A (optional) dependency doesn’t have a distribution on conda however it is added as an optional dependency in our PyPi release. And in the recipe one of test command uses the package (nflows). In order to keep everything coherent as much as possible I replaced the URL with the current upstream head. But I guess I can still keep the PyPi url and just comment out the check for now. I’ll look into it later. Thanks for the “pasta” though :)